medical_physician_us/models/medical_physician.py

```diff
@@ -28,7 +28,8 @@ class MedicalPhysician(models.Model):
         inverse=lambda s: s._inverse_identification(
             'dea_num', 'DEA',
         ),
-        help='Drug Enforcement Agency #',
+        help='''Drug Enforcement Agency # - \
```

Comment:
Best to break this string off. Current strategy is adding a bunch of blank whitespace.

```
help='Drug Enforcement Agency # - '
     'two alpha characters and seven digits'
```